docusaurus/docs/create-a-plugin/develop-a-plugin/best-practices.md

```diff
@@ -38,15 +40,14 @@ Is something missing from this list? [Let us know](https://github.com/grafana/pl
 - **Add query editor help.** Query editors can be complex and it's useful to provide help to users. For more information, refer to [Add query editor help](/create-a-plugin/extend-a-plugin/add-query-editor-help).
 - **Skip hidden or empty queries.** This avoids executing unnecessary or wrong queries. Data sources implementing [`DataSourceWithBackend`](https://github.com/grafana/grafana-plugin-examples/blob/0532f8b23645251997088ac7a1707a72d3fd9248/examples/datasource-basic/src/datasource.ts#L5) only need to implement the method `filterQuery`. Refer to this [example](https://github.com/grafana/grafana/blob/fd5f66083c91b9759ae7772f99b80c9342b93290/public/app/plugins/datasource/loki/datasource.ts#L1085).
 - **Specify a default query.** Default queries can help users to discover how queries are written for the plugin. Refer to this [example](https://github.com/grafana/grafana-plugin-examples/blob/0532f8b/examples/datasource-streaming-backend-websocket/streaming-backend-websocket-plugin/src/datasource.ts#L39-L41).
-- **Avoid using `console.log` in your plugin.** Console messages are usually for debugging purposes and therefore not suitable to ship to the client.
 
 ### Frontend (only) plugins
 
 - **Data sources running only on the frontend typically use the [Grafana proxy](/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins#add-a-proxy-route-to-your-plugin) to access an external service.** This s a simple way of adding support for queries in your plugin, and it doesn't require Golang knowledge. However, there are use cases for which writing a backend plugin is necessary. Refer to [Backend plugins](/introduction/backend-plugins#use-cases-for-implementing-a-backend-plugin) for more information about those.
```

Comment:
This s a
to
This is a

Reply:
I'm not following this suggestion. Can you please elaborate?

Reply:
Ah, when I was adding the comment it looked like it would be on the line with the issue.

On line 46, it says "This s a" it should be "This is a" (missing an "i")
On line 68, it says, "that be used" it should be "that can be used" (missing "can")